app/helpers/audits_helper.rb

```diff
@@ -136,6 +136,10 @@ def audited_type(audit)
                     "#{audit.associated_type || 'Global'}-#{type_name}"
                   when 'LookupKey'
                     'Smart Variable'
```

Comment:
Keeping this because of old audits that don't have a connected lookup key and therefore I can't determine their type.